games-simulation/singularity/singularity-1.00.recipe

```diff
@@ -5,11 +5,15 @@ have a chance."
 HOMEPAGE="https://singularity.github.io/"
 COPYRIGHT="2005-2023 EMH Software, Singularity community"
 LICENSE="GNU GPL v2"
-REVISION="2"
+REVISION="3"
 srcGitTag="1.00"
+musicVersion="007"
 SOURCE_URI="https://github.com/singularity/singularity/archive/v$srcGitTag.tar.gz"
+SOURCE_URI_2="https://www.emhsoft.com/singularity/endgame-singularity-music-007.zip"
```

Comment:
Use musicVersion here too :)
```diff
-SOURCE_URI_2="https://www.emhsoft.com/singularity/endgame-singularity-music-007.zip"
+SOURCE_URI_2="https://www.emhsoft.com/singularity/endgame-singularity-music-$musicVersion.zip"
```

Reply:
Just saw this, srcGitTag is the same as portVersion, maybe use that and drop srcGitTag?